src/main/java/org/smartregister/pathevaluator/condition/ConditionHelper.java

```diff
@@ -54,7 +54,7 @@ public boolean evaluateActionConditions(DomainResource target, Action action, St
 				List<Resource> resources = (List<Resource>) actionHelper.getConditionSubjectResources(condition, action,
 				    target, planIdentifier);
 				if (resources != null && !resources.isEmpty()) {
-					target=target.toBuilder().contained(resources).build();	
+//					target=target.toBuilder().contained(resources).build();	// TODO : FIX
```

Comment:
This cannot be removed

Reply:
check if instance of target is domain resource

Reply:
I have changed the method signature which was earlier accepting target as DomainResource. @githengi

Reply:
that is okay however contained cannot be removed as alot of plans will be affected. So here you can check is target is instance of Domain resource i.e the below example
```
if (resources != null && !resources.isEmpty() && target instanceof DomainResource) {
	target=((DomainResource)target).toBuilder().contained(resources).build();	
}
```

Reply:
Okay, I will test with this approach.
Are we using the same Bundle FHIR resource @githengi ?

Reply:
For now, yes its its a bundle you can add the resources to the bundle if its not an instance of DomainResource